### PR TITLE
LPS-178665 Use getLocalizableLabel function to get the titleFieldId label

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-178665

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/useObjectDetailsForm.ts [hunk 1]
```diff
@@ -73,15 +74,23 @@ export function useObjectDetailsForm({
...
 		const errors: ObjectDetailsErrors = {};
 
-		const label = objectDefinition.label?.[defaultLanguageId];
-
-		if (invalidateRequired(label)) {
+		if (
+			invalidateRequired(
+				getLocalizableLabel(
+					objectDefinition.defaultLanguageId as Liferay.Language.Locale,
+					objectDefinition.pluralLabel
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/useObjectDetailsForm.ts:81]: @danvb112 Here we should use `objectDefinition.label`.

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by danvb112** [comment]
ci:test:sf

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
The failures don't seem related to the changes.

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**